tests/test_models.py

```diff
@@ -186,7 +381,10 @@ def test_schema_loads(self, schema_klass, expected_klass, instance_data):
         THEN the extra field should be excluded
         AND the result should be an instance of the expected class
         """
-        schema = schema_klass()
+        if callable(schema_klass):
+            schema = schema_klass()
+        else:
+            schema = schema_klass
```

Comment:
There is some kind of undocumented standard to declare our models and their schema. It looks like this:

```python
@dataclass
class Foo(Base, FromDictMixin):
    # fields


FooSchema = cast(
    Type[BaseSchema], marshmallow_dataclass.class_schema(Foo, BaseSchema)
)

Foo.SCHEMA = FooSchema()
```

If you do this for the schemas you are testing here, you won't need these changes and the code will be a little more consistent.

Reply:
Hum I was not entirely sure I understood your suggestion but I think successfully applied it ? :sweat_smile: